eng/pipelines/templates/scripts/verify-path-length.yml

```diff
@@ -9,7 +10,9 @@ steps:
       scriptSource: inline
       script: |
         # Verifies Length of file path for all files in the SourceDirectory.
-        # Breaks if file path is 260 characters or more
+        # File paths and directory paths must be less than 260 and 248 characters respectively on windows OS
+        # Repo users get 49 characters for the repo clone path
+        # Script makes sure that paths in the repo are less than 210 and 198 for files and directories respectively.
```

Comment:
You should update these comments now given the base length is parameterized.

Reply:
Updated